### ISSUE TITLE
Convert MD->PDF alt text doesn't display on image

### ISSUE THREAD

**Issue by slarge** [andybeet/ESR#1]

**Comment by andybeet** [comment]
Alt text now works for html and md versions of the document when moused over.

Still missing after pdf conversion. (According to Gail, this isn't a sign of non compliance, but more of a user defined option within acrobat). However no sign of alt text being listed in pdf metadata

**Comment by andybeet** [comment]
Found a way (thanks to @sgaichas) to get "alt text" to appear on a mouse over for pdfs (using [tooltips](http://www.richschwinn.com/richschwinn/accessibility/)) but this doesn't pass accessibility checks in acrobat.

**Comment by andybeet** [comment]
resolved: created a rmd from content (using fig.cap="alt txt") => convert to word (using a template for headings and image centering) => convert to pdf.

All alt text are preserved
